internal/actions/check.go

```diff
@@ -33,7 +34,12 @@ func CheckForMultipleWatchtowerInstances(client container.Client, cleanup bool)
 		return nil
 	}
 
-	log.Info("Found multiple running watchtower instances. Cleaning up.")
+	if scopeUID != "" {
```

Comment:
Instead of actually cleaning a running instance, would it be ok if it just didn't start? The implementation looks simpler this way.

Reply:
That would disable the auto-update feature of watchtower. I think the way to approach this is that, if scope is set, send a filter which also contains the scope on the line below, and then just proceed as usual

```
  containers, err := client.ListContainers(filters.WatchtowerContainersFilter)
```

Reply:
Changed the filter from
```
client.ListContainers(filters.WatchtowerContainersFilter)
```
to
```
client.ListContainers(filters.FilterByScope(scope, filters.WatchtowerContainersFilter))
```
But, because of actual filtering is needed to test the logic of this function (and the mocked [`ListContainers`](https://github.com/containrrr/watchtower/blob/f76c48a95ec0bb61d84080a3546d529d8ea3f839/internal/actions/mocks/client.go#L43) simply returns all of the running containers. PS: changing this behavior causes multiple fatal failures in tests), I removed the test I had written in `actions_suite_test.go`.

But, since `FilterByScope` is being tested in `filters_test.go`, everything should be fine. But, in the future, we might need to put some effort in making a working mock of `ListContainers`.